PyDSS/dataset_buffer.py

```diff
@@ -143,6 +145,10 @@ def max_num_bytes(self):
 
     def write_value(self, value):
         """Write the value to the internal buffer, flushing when full."""
+        print(value)
+        if pd.isna(value[0]):
+            value = [-1 for i in value]
```

Comment:
What’s going on here?

Reply:
reverting back. This had to do with non-convergence

Reply:
I think this will break some things. We store `np.NaN` whenever there is a convergence error. HDF5 converts that to a sentinel value on writes. It converts those back to `np.Nan` on reads. Do we need to debug some error condition?

Reply:
reverting back. All tests seem to pass.